doc/wiki/dashboard-audit.md

```diff
@@ -0,0 +1,668 @@
+
+*Sample SQL query*
+```sql
+SELECT "blood_pressures_per_facility_per_days"."facility_id"
+FROM "blood_pressures_per_facility_per_days"
+WHERE "blood_pressures_per_facility_per_days"."deleted_at" IS NULL
+  AND "blood_pressures_per_facility_per_days"."facility_id" IN
+    (SELECT "facilities"."id"
+     FROM "facilities"
+     WHERE "facilities"."deleted_at" IS NULL
+       AND "facilities"."id" IN
+         (SELECT "facilities"."id"
+          FROM (
+                  (SELECT "facilities".*
+                   FROM "facilities"
+                   WHERE "facilities"."deleted_at" IS NULL)
+                UNION
+                  (SELECT "facilities".*
+                   FROM "facilities"
+                   WHERE "facilities"."deleted_at" IS NULL
+                     AND "facilities"."facility_group_id" IN
+                       (SELECT id
+                        FROM (
+                                (SELECT "facility_groups".*
+                                 FROM "facility_groups"
+                                 WHERE "facility_groups"."deleted_at" IS NULL
+                                   AND "facility_groups"."deleted_at" IS NULL)
+                              UNION
+                                (SELECT "facility_groups".*
+                                 FROM "facility_groups"
+                                 WHERE "facility_groups"."deleted_at" IS NULL
+                                   AND "facility_groups"."deleted_at" IS NULL
+                                   AND "facility_groups"."organization_id" IN
+                                     (SELECT "organizations"."id"
+                                      FROM "organizations"
+                                      WHERE "organizations"."deleted_at" IS NULL))) "facility_groups"))) "facilities"))
+  AND (((YEAR,
+         DAY) IN (('2021',
+                   '91'),('2021',
+                          '90'),('2021',
+                                 '89'),('2021',
+                                        '88'),('2021',
+                                               '87'),('2021',
+                                                      '86'),('2021',
+                                                             '85'))))
+GROUP BY "blood_pressures_per_facility_per_days"."facility_id"
+HAVING (SUM(bp_count) >= 10)
+```
+
+**Where it's shown in the Dashboard**
++ **Home:** "Facilities using Simple" and "Inactive facilities" cards
+
+**Things we should fix**
++ [X] Update "Facilities using Simple" subtitle to be "Inactive facilities <10 patients with BPs recorded in the last 7 days"
+
+---
+
+## BP measure taken
+**Copy used in the Dashboard**
+All BP measures taken in a month by healthcare workers at `region_name`
+
+**How it's calculated**
+Counts all blood pressures recorded by each user at a facility where the patient:
++ Is hypertensive
```

Comment:
Seems like it should be all patients, not just hypertensive patients. Maybe I don't understand the business purpose here.

Reply:
The dashboards should be showing hypertensive only everywhere...that is one of the default reporting criteria.